src/http-gateways/trustless-gateway.md

```diff
@@ -183,6 +183,22 @@ returned:
     returned to the client, the HTTP status code has already been sent to the
     client.
 
+### :dfn[skip-raw-blocks] (request query parameter)
```

Comment:
:information_source:  `skip-raw-blocks` is now a URL query parameter 
(rationale explained in "Alternatives" → "Why not CAR content type parameter" section)